modules/document_repository/php/document_repository.class.inc

```diff
@@ -79,9 +79,9 @@ class Document_Repository extends \NDB_Menu_Filter
                           'v.Date_uploaded',
                          );
 
-        $openAccordion = $_GET['openAccordion'];
+        $openAccordion = $_GET['openAccordion'] ?? '';
         $this->tpl_data['openAccordion'] = $openAccordion;
-        $filtered = $_GET['filtered'];
+        $filtered = $_GET['filtered'] ?? '';
```

Comment:
Both of these variables are used as booleans in the template, so this line and 82 should be `?? false;` instead.

Reply:
In that case it should probably be using a ternary, not a null coalesce, since `$_GET['filtered']` is a string value.